wunderio/next4drupal-project
# Nex 2 monorepo

*Link to ticket: [NEX-2 monorepo](https://wunder.atlassian.net/jira/software/c/projects/NEX/boards/389?modal=detail&selectedIssue=NEX-2)*

*Changes proposed in this PR:*
- Drupal related files moved to /drupal folder
- Project renamed to next4drupal-project

*How to test:*
1. Check no files are missing and no secret files are commited
2. Check lando.yml is correct (renaming of the project is successfully and fully done)
3. Locally checkout to the branch and run the following commands. Check that site is installed.

- lando start
- composer install
- import db
- drush cmi
- drush cr
- lando drush ucrt wunder with email and pwd (check ucrt command syntax)
- lando drush user-add-role "next_js_admin" wunder
- go to /admin/config/services
https://next4drupal-project.lndo.site/admin/config/people/simple_oauth
Generate keys to ../oath (folder /oath should be under /drupal but above /web)
- Go to 
go to /admin/config/services/consumer and create wunder consumer with next js admin role
- copy consumer uuid to .env.local in /next
- try out with creating a drupal article and viewing it in drupal and previewing in nextjs.
For that, in /next run 'npm run dev' and go to check if localhost:3000 works ok and shows your new article.
then in drupal View your article and open for preview in next.js

*Link to feature environment:*
<add silta url here>
